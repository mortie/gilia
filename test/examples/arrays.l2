arr := [10 20 30]
print arr
print arr.2

arr.1 = "nope"
print arr
