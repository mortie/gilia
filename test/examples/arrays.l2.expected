[10 20 30]
30
[10 nope 30]
